jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -12,16 +12,18 @@ else
 fi
 echo  "{'REPO': '$REPOSITORY_FILE_URL', 'BUILD_TIMESTAMP': '${BUILD_TIMESTAMP}', 'BUILD_LOG': '$BUILD_URL', 'BUILD_ID': $BUILD_JOB_NUMBER, 'BUILD_STATUS': '$BUILD_STATUS'}" > STATUS
 
-alias rsync_upload="rsync -e \
-    'ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa' \
```

Comment:
All variables come from the environment. Some are unique to the Jenkins instance (HOME), some come from job parameters (UPLOAD_SERVER_USER_NAME) and some come from plugins. I don't think documenting every variable is worth the trouble.